<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Element is present]]>
    </line>
  </reporter-output>
  <suite name="Suite" duration-ms="41600" started-at="2018-06-03T06:36:53Z" finished-at="2018-06-03T06:37:34Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="41600" started-at="2018-06-03T06:36:53Z" finished-at="2018-06-03T06:37:34Z">
      <class name="testScript.Test1">
        <test-method status="PASS" signature="launchBrowser()[pri:0, instance:testScript.Test1@57baeedf]" name="launchBrowser" is-config="true" duration-ms="8660" started-at="2018-06-03T12:06:53Z" finished-at="2018-06-03T12:07:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launchBrowser -->
        <test-method status="FAIL" signature="testFaceBookHkome()[pri:0, instance:testScript.Test1@57baeedf]" name="testFaceBookHkome" duration-ms="30803" started-at="2018-06-03T12:07:01Z" finished-at="2018-06-03T12:07:32Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.fail(Assert.java:100)
	at testScript.Test1.testFaceBookHkome(Test1.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Element is present]]>
            </line>
          </reporter-output>
        </test-method> <!-- testFaceBookHkome -->
        <test-method status="PASS" signature="closeBrowser()[pri:0, instance:testScript.Test1@57baeedf]" name="closeBrowser" is-config="true" duration-ms="2018" started-at="2018-06-03T12:07:32Z" finished-at="2018-06-03T12:07:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- testScript.Test1 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
